spec/controllers/companies_controller_spec.rb

```diff
@@ -13,9 +13,9 @@
                       hiring: 1,
                   hire_count: 5
                        }
```

Comment:
The association between users and companies isn't built into the factories so I created the user separately. If this becomes a burden later, I'll add the association to the factory.